### ISSUE TITLE
Python 3

### ISSUE THREAD

**Issue by elleven11** [hugsy/gef#612]
Your issue will be closed unless you confirm the following (insert `x` in the brackets if done/agreed):
  * [X ] Did you use the latest version of GEF from `master` branch?
  * [X ] Did you read the [documentation](https://gef.readthedocs.org/en/latest/) first?
  * [ X] Did you check [issues](https://github.com/hugsy/gef/issues) (including the closed ones) - and the [PR](https://github.com/hugsy/gef/pulls)?

### Step 1: Describe your environment

  * Operating System / Distribution: Arch Linux
  * Architecture: amd64
  * GDB version (including the Python library version) copy the output of `$ gdb -ex "pi import sys;print(gdb.VERSION);print(sys.version)"`: 
10.1
3.9.1 (default, Feb  6 2021, 06:49:13)
[GCC 10.2.0]

#### Steps to reproduce

  1. Python 3.9
  2. shellcode get id

**The more info is provided, the more likely your issue will be confirmed and handled**

Long story short, the shellcode get command tries to execute HTMLParser with the unescape attribute:
[!] Command 'shellcode get' failed to execute properly, reason: 'HTMLParser' object has no attribute 'unescape'

in Python 3.9 this attribute is no longer available (also was deprecated since 3.4). To fix this issue i had to downgrade to python 3.8
it would be nice if you guys used something else other than HTMLParser to grab the shellcode.

**Comment by hugsy** [comment]
Fixed
